Makefile

```diff
@@ -1,18 +1,16 @@
-.PHONY: image clean cleaner cleanest mrproper
+PHONY: image clean cleaner cleanest mrproper
 
```

Comment:
Are you certain?

Reply:
That is an unintentional change.  I'll fix it.